zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -32,10 +34,26 @@
       return getContactAdminForm().find('#contact-admin-send-button');
     }
 
+    // Returns a span.
+    function getContactAdminMessageErrorField(){
+      return getContactAdminForm().find('#contactAdminForm\\:messageField\\:input\\:contact-admin-error-message\\:contactAdminForm\\:messageField\\:input\\:contact-admin-message');
+    }
+
     function toggleSendButton() {
       var hasMessage = getContactAdminMessageField().val() !== '';
       getContactAdminSendButton().prop('disabled', !hasMessage);
     }
+
+    function completeSending() {
+      var errMsg = getContactAdminMessageErrorField().text();
+      if (errMsg != '') {
```

Comment:
Instead of checking the text, can you only check if the error field is displayed or not?